src/interfaces/nav-controller-interface.ts

```diff
@@ -0,0 +1,31 @@
+import { LitElement } from 'lit-element';
```

Comment:
Not sure if you want to tackle this now or not, but I would upgrade to lit2 when you can.